src/report-links/report-links.tsx

```diff
@@ -1,181 +1,96 @@
 import React, { useEffect, useState } from "react";
-import { openmrsFetch } from "@openmrs/esm-api";
 import { Trans } from "react-i18next";
 
 import { CommonWidgetProps, LoadingStatus } from "../models";
 import WidgetHeader from "../commons/widget-header/widget-header.component";
 import resources from "./translations";
 import { initI18n } from "../utils/translations";
+import ChartLoader from "../charts/chart-loader.component";
 
 import globalStyles from "../global.css";
 import styles from "./report-links.css";
+import ReactModal from "react-modal";
+
+ReactModal.defaultStyles.overlay.backgroundColor = "#ffffff00";
+const customStyles = {
```

Comment:
is it modal's custom styles? if Yes, can this be named like
```suggestion
const modalStyles = {
```
Else we can prefer getting the styles from css class if possible.